frontend/src/methods/index.ts

```diff
@@ -19,3 +20,50 @@ export const getServiceHealthStatus = (service) => {
     ? TCommonStatuses.READY
     : TCommonStatuses.UNHEALTHY;
 };
+export const sortServers = (items) => {
+ const result = items.reduce((acum,reducer, index) => {
+   if (items[index - 1]) {
+    return reducer["metadata"]["creationTimestamp"] !== items[index - 1]["metadata"]["creationTimestamp"] ? acum = true : acum = false
+  }else  return false
+  }, false)
+  return result
+}
+export  const sortServersByTimestamp = (items, status) => {
+  return status
+    ? items.sort((a: any, b: any): any => {
+        return DateTime.fromISO(a["metadata"]["creationTimestamp"]).ts <
+          DateTime.fromISO(b["metadata"]["creationTimestamp"]).ts
+          ? 1
+          : -1;
+      })
+    : items.sort((a: any, b: any): any => {
+        return DateTime.fromISO(a["metadata"]["creationTimestamp"]).ts >
+          DateTime.fromISO(b["metadata"]["creationTimestamp"]).ts
+          ? 1
+          : -1;
+      });
+};
+export const sortServersByNames = (items,status) => {
+  return status
+    ? items.sort(function(a, b){
+      if(a["metadata"]["uid"] < b["metadata"]["uid"]) { return -1; }
+      if(["metadata"]["uid"] > b["metadata"]["uid"]) { return 1; }
```

Comment:
looks like there's a typo here... should be `a["metadata"]["uid"]`

Reply:
Fixed.